cpp/include/cudf/aggregation.hpp

```diff
@@ -67,6 +67,7 @@ class aggregation {
     ALL,             ///< all reduction
     SUM_OF_SQUARES,  ///< sum of squares reduction
     MEAN,            ///< arithmetic mean reduction
+    M2,              ///< groupwise sum of squares of differences from the group mean
```

Comment:
This list of aggregations is shared among multiple algorithms, so we shouldn't refer to just "groupwise". I see the enums below have the same problem. 
```suggestion
    M2,              ///< sum of squares of differences from the mean
```